prime-dotnet-webapi/PrimeEnvironment.cs

```diff
@@ -109,5 +109,27 @@ public static class LdapApi
         {
             public static readonly string Url = GetEnvironmentVariable("LDAP_API_URL") ?? "https://common-logon-dev.hlth.gov.bc.ca/ldap";
         }
+
+        public static class PrimeSentryKeys
+        {
+            public static readonly string DevEnvDsn = "https://b10d3b20d6cc4eb39e0d90997c72fd0c@o988003.ingest.sentry.io/5945123";
+
```

Comment:
How to test, that is, know that Sentry is capturing the logs?

Reply:
We can check the Sentry dashboard (will need to add everyone there), I think in the future it will be integrated with a slack channel too. For development I just check the dashboard and my email as when I was setting it up I wanted to see how everything is being sent and whether or not they were being sent. We can also enable `sentryOptions.Debug` and that will print to debug console when an exception is successfully sent to Sentry or if something went wrong but I disabled that option to reduce console noise.

Reply:
Is it convenient to add us to the Dev Sentry dashboard on a by-request basis?

Reply:
It is very easy to add you to the dashboard, I think the plan was to add the team at some point, I just added you, check your email. :)

Reply:
@AMiKaH I got an email today from Sentry about "Trial Ending Soon" ... what are plans beyond the trial?

Reply:
Yeah I think all members got that email. The trial period started after the first invite that I sent. There's a free opensource version that we can see if it can work for our set up. My understanding is that paid version has more features about performance monitoring but nothing additional for exceptions.

Reply:
It seems that the PR should then be working with the free Sentry version or else MoH needs to know and be willing to pay the cost of the non-free version

Reply:
The PR should work on free version. I am not sure why it triggered the trial period, it started when I invited Jason as a "Billing" role so maybe it saw Billing and thought that we will pay lol.